CHANGES.md

```diff
@@ -10,6 +10,15 @@ creating a new release entry be sure to copy & paste the span tag with the
 updated. Only the first match gets replaced, so it's fine to leave the old
 ones in. -->
 
+-------------------------------------------------------------------------------
+## __cylc-8.1.3 (<span actions:bind='release-date'>Coming Soon</span>)__
+
+### Fixes
+
+[#5386](https://github.com/cylc/cylc-flow/pull/5386) Fix bug where
+absence of `job name length maximum` in PBS platform settings would cause
+Cylc to crash when preparing the job script.
```

Comment:
This ought to be conflicting. The fact it's not suggests that it's going to duplicate the 8.1.3 section when merged. Might need to merge 8.1.x into this branch and sort it out